### PR TITLE
updates to codebase for python 3

### PR DESCRIPTION
**Pull request by marksmayo**
- removed unnecessary inheritance from object
 - removed unnecessary u prefix for strings
 - sorted some imports
- converted strings into f-strings
- simplified a conditional

### CODE REVIEW

File: script/release.py [hunk 1]
```diff
@@ -51,72 +51,72 @@ def get_new_version():
 
 def update_version_file(version):
     new_version = VERSION_FORMAT.format(version)
-    with open(VERSION_FILE_PATH, 'w') as f:
+    with open(VERSION_FILE_PATH, 'w', encoding="utf-8") as f:
```
Comment (mykola-mokhnach) [inline script/release.py:54]: Please use single quotes everywhere

**Comment by mykola-mokhnach** [comment]
Could you please make sure the CI is ok?

File: test/functional/ios/helper/test_helper.py [hunk 2]
```diff
@@ -23,7 +23,7 @@
 from . import desired_capabilities
 
 
-class BaseTestCase(object):
+class BaseTestCase():
```
Comment (KazuCocoa) [inline test/functional/ios/helper/test_helper.py:26]: `()` also can be removed

**Comment by marksmayo** [comment]
> Could you please make sure the CI is ok?

the last two tests have been running for freaking ages, but looks like it's sorted now

**Comment by KazuCocoa** [comment]
I've kicked all of CI tasks

**Pull request closed without merging**